src/constants/range.js

```diff
@@ -23,7 +23,7 @@ export const DEFAULT_RANGES_STATE = {
 
 export const SPLIT_RANGE_AT = 'SPLIT_RANGE_AT';
 export const GROUP_RANGES = 'GROUP_RANGES';
-export const ON_SELECT_RANGE = 'ON_SELECT_RANGE';
+export const SELECT_RANGE = 'SELECT_RANGE';
 export const UPDATE_RANGE = 'UPDATE_RANGE';
 export const MOVE_POINT = 'MOVE_POINT';
 export const DELETE_RAGE = 'DELETE_RAGE';
```

Comment:
I know its not edited in this PR, but think this may be a typo! 😄

Reply:
Action names doesn't include 'ON' in this project, I've just made it the same as the others